azure/notebooks/kubernetes/Dask_Kube_XGBoost.ipynb

```diff
@@ -0,0 +1,917 @@
+{
```

Comment:
<p>Remove vanilla</p><p> </p><p>*Forest Inference Library (FIL)</p><p> </p><p>Let's remove the additional content that goes into too much detail of XGB implementation (according to the official documentation ...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Sounds good. </p>